modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -1753,15 +1753,15 @@ def test_reset_index_with_named_index(
     if test_async_reset_index:
         # The change in index is not automatically handled by Modin. See #3941.
         modin_df.index = modin_df.index
-        modin_df._to_pandas()
+        modin_df.modin.to_pandas()
```

Comment:
In several places I use the public method to test its functioning.

Reply:
Maybe we should use the public method everywhere to not mix up the code?

Reply:
ping

Reply:
Let's do it after https://github.com/modin-project/modin/pull/7014#discussion_r1514473560